lib/rdf.js

```diff
@@ -64,6 +75,15 @@ function createRDF(manifest) {
     delete jetpackMeta["em:icon64URL"];
   }
 
+  if (needsBootstrapJS) {
```

Comment:
When ```bootstrap.js``` is not provided by the addon developer, [the default ```bootstrap.js```](https://github.com/mozilla-jetpack/jpm-core/blob/master/data/bootstrap.js) is used. With this ```bootstrap.js```, the addon's ```resource://``` URI registration occurs in [resource://gre/modules/commonjs/sdk/addon/bootstrap.js](https://hg.mozilla.org/mozilla-central/file/c59c022943f6/addon-sdk/source/lib/sdk/addon/bootstrap.js#l101). If a custom ```bootstrap.js``` is provided, the program flow mentioned above may not occur, so the real icon URLs are unknow.